<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QWidget" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>395</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Loan amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loan_amount">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignHCenter';
  border-radius: 10px;
}

background-color : white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loan_term">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>55</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignHCenter';
  border-radius: 10px;
}

background-color : white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>395</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Loan term</string>
   </property>
  </widget>
  <widget class="QComboBox" name="month_year">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>55</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Months</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Years</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="int_rate">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignHCenter';
  border-radius: 10px;
}

background-color : white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>395</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Interest rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>145</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Payment type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="payment_type">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>145</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentText">
    <string>Annuity</string>
   </property>
   <item>
    <property name="text">
     <string>Annuity</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Differentiated</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="calculate">
   <property name="geometry">
    <rect>
     <x>243</x>
     <y>200</y>
     <width>222</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: #acd4b4;
  color: white; 
  border-radius: 20px;
}

QPushButton:pressed {
    background-color: #8fba8f;
}</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>280</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="month_payment">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignHCenter';
  border-radius: 10px;
}

background-color : white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>445</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Monthly payment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>325</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accured_inter">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>325</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignHCenter';
  border-radius: 10px;
}

background-color : white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>325</y>
     <width>445</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Accrued interest</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>370</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="total_cost">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
  qproperty-alignment: 'AlignVCenter | AlignHCenter';
  border-radius: 10px;
}

background-color : white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>445</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string> Total cost of the loan</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="pay_sch">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>41</y>
     <width>441</width>
     <height>365</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>5</y>
     <width>441</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignHCenter';
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>Payment schedule (differentiated type only)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>380</y>
     <width>441</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignHCenter';
  background-color: #dbdbdb;
border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>222</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SB Sans Text</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: #acd4b4;
  color: white; 
  border-radius: 20px;
}

QPushButton:pressed {
    background-color: #8fba8f;
}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <zorder>label_12</zorder>
  <zorder>label_10</zorder>
  <zorder>label_8</zorder>
  <zorder>label_5</zorder>
  <zorder>label_4</zorder>
  <zorder>label</zorder>
  <zorder>loan_amount</zorder>
  <zorder>label_2</zorder>
  <zorder>loan_term</zorder>
  <zorder>month_year</zorder>
  <zorder>label_3</zorder>
  <zorder>int_rate</zorder>
  <zorder>label_6</zorder>
  <zorder>payment_type</zorder>
  <zorder>calculate</zorder>
  <zorder>label_7</zorder>
  <zorder>month_payment</zorder>
  <zorder>label_9</zorder>
  <zorder>accured_inter</zorder>
  <zorder>label_11</zorder>
  <zorder>total_cost</zorder>
  <zorder>line</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>pay_sch</zorder>
  <zorder>clear</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
